src/course-unit/data/slice.js

```diff
@@ -115,6 +116,9 @@ const slice = createSlice({
       // This avoids the need to copy the array beforehand
       state.courseVerticalChildren.children.sort((a, b) => (indexMap.get(a.id) || 0) - (indexMap.get(b.id) || 0));
     },
+    fetchXBlockIframeResources: (state, { payload }) => {
+      state.xblockIframeHtmlAndResources.push(payload);
```

Comment:
```suggestion
      state.xblockIFrameHtmlAndResources.push(payload);
```

Reply:
And in all other cases either `iframe` or `somethingIFrame`

Reply:
Done